README.md

```diff
@@ -18,6 +18,12 @@ Parameters in some Making Tax Digital for Income Tax APIs map to box numbers in
 
 ---
 
+### 9 January 2025
+
+#### SELF ASSESSMENT INDIVIDUAL DETAILS API
```

Comment:
Comparing to other entries, headings aren't uppercase & `The documentation for version 1.0 has also been removed.` is true for this as well.